### ISSUE TITLE
Broken table value in latest 2.27.0 version

### ISSUE THREAD

**Issue by krenovut** [simplygoodsoftware/pyrusapi-python#83]
Simple code for looking field's attributes:

```
  for field in task.flat_fields:
        if "Программа " not in field.name:
            continue

        print(vars(field))
```

in version 2.26.0 :
`{'id': 103, 'type': 'table', 'name': 'Программа 2', 'value': [<pyrus.models.entities.TableRow object at 0x000002D270B5A1B0>], 'parent_id': 94}`    

in version 2.27.0
`{'id': 103, 'type': 'table', 'name': 'Программа 2', 'value': [], 'parent_id': 94}`

Fix this please.

**Comment by krenovut** [comment]
Possible fix:
Replace the Table class code with
```
class Table(list):
    """
        Value of FormField table
        List of `models.entities.TableRow`
    """

    def __init__(self, *args):
        list.__init__(self)
        self.extend([TableRow(**value) for value in args]  #!
```
as the assigment to self are not allowed in python.

**Comment by dkuptsov** [comment]
Hello! Thank you for your contribution to improving the library. This fix was released in version 2.27.1
